### PR TITLE
Setup series repo

### PR DESCRIPTION
**Pull request by VitaliyaIoffe**
1) Publishing LIVE packages is no longer supported 

According to the new release policy live packages are not allowed.
Only releases and pre-releases would be published to the S3.

Needs for: #6185

2) Publishing packages to series repo 

New release policy provides new rules for publishing packages.
Now, only major version is defining bucket repo, for more convenience
use next naming series-<MAJOR_VERSION>.

Needs for: #6185

3) Enable publish pre-release tags 

Closed: #6185

### CODE REVIEW

File: .gitlab.mk [hunk 1]
```diff
@@ -117,7 +117,7 @@ vms_shutdown:
 GIT_DESCRIBE=$(shell git describe HEAD)
 MAJOR_VERSION=$(word 1,$(subst ., ,$(GIT_DESCRIBE)))
 MINOR_VERSION=$(word 2,$(subst ., ,$(GIT_DESCRIBE)))
```
Comment (VitaliyaIoffe) [inline .gitlab.mk:119]: Do we need `MINOR_VERSION` here?

File: .github/workflows/centos_7.yml [hunk 2]
```diff
@@ -37,8 +37,8 @@ jobs:
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           AWS_S3_ENDPOINT_URL: ${{ secrets.AWS_S3_ENDPOINT_URL }}
-          LIVE_REPO_S3_DIR: ${{ secrets.LIVE_REPO_S3_DIR }}
           RELEASE_REPO_S3_DIR: ${{ secrets.RELEASE_REPO_S3_DIR }}
+          PRERELEASE_REPO_S3_DIR: ${{ secrets.PRERELEASE_REPO_S3_DIR }}
```
Comment (VitaliyaIoffe) [inline .github/workflows/centos_7.yml]: Have to add `secrets.PRERELEASE_REPO_S3_DIR` in repo secrets

Comment (VitaliyaIoffe) [reply .github/workflows/centos_7.yml]: Have added `secrets.PRERELEASE_REPO_S3_DIR`. 
It was tested in 
https://github.com/tarantool/tarantool/runs/3329058073?check_suite_focus=true

**Comment by Totktonada** [comment]
For the history: the new release policy document is [here](https://github.com/tarantool/tarantool/discussions/6182).

File: .gitlab.mk [hunk 3]
```diff
@@ -117,7 +117,7 @@ vms_shutdown:
...
-BUCKET="$(MAJOR_VERSION).$(MINOR_VERSION)"
+BUCKET="series-$(MAJOR_VERSION)"
```
Comment (Totktonada) [inline .gitlab.mk:120]: I propose the flat layout for tarballs, without splitting for the series: `https://download.tarantool.org/tarantool/src/*.tar.gz`. I would even copy older tarballs there for convenience of ebuild / pkgbuild / port makefiles writters.

I'll add this point into the release policy RFC.

File: .gitlab.mk (hunk 1 above)
Comment (Totktonada) [inline .gitlab.mk:119]: It is not used anywhere anymore.

File: .gitlab.mk [hunk 4]
```diff
@@ -134,6 +134,10 @@ deploy: export LD_LIBRARY_PATH=/usr/local/lib
 deploy:
 	echo ${GPG_SECRET_KEY} | base64 -d | gpg --batch --import || true
 	case "${GITHUB_REF}" in                                       \
+	refs/tags/*-alpha*|refs/tags/*-beta*|refs/tags/*-rc*) \
```
Comment (Totktonada) [inline .gitlab.mk:137]: The commit message contains a mention of the `*-dev*` tag. It looks as a mistake.

**Review by Totktonada — APPROVED** [review]
Except the source tarballs layout (see above) everything looks okay for me. I'll approve beforehand, but I expect that this point will be changed prior to merging this PR.

I would highlight that the final decision about disabling per push deployments to the live repository is somewhere between @kyukhin and @Mons (my opinion is that we should disable it and it is mentioned in the policy document).

NB: Don't forget about secret variables for [arm64 packaging workflows](https://github.com/tarantool/tarantool/pull/6269) when this PR will land.

File: .gitlab.mk (hunk 3 above)
Comment (VitaliyaIoffe) [reply .gitlab.mk:119]: Fixed

File: .github/workflows/centos_7.yml (hunk 2 above)
Comment (VitaliyaIoffe) [reply .github/workflows/centos_7.yml:41]: Done

File: .gitlab.mk (hunk 1 above)
Comment (VitaliyaIoffe) [reply .gitlab.mk]: Removed

File: .gitlab.mk (hunk 4 above)
Comment (VitaliyaIoffe) [reply .gitlab.mk:136]: Fixed

**Comment by VitaliyaIoffe** [comment]
> Except the source tarballs layout (see above) everything looks okay for me. I'll approve beforehand, but I expect that this point will be changed prior to merging this PR.
> 
> I would highlight that the final decision about disabling per push deployments to the live repository is somewhere between @kyukhin and @Mons (my opinion is that we should disable it and it is mentioned in the policy document).
> 
> NB: Don't forget about secret variables for [arm64 packaging workflows](https://github.com/tarantool/tarantool/pull/6269) when this PR will land.

All was fixed and secret for `pre-release repo` was added.

**Review by kyukhin — APPROVED** [review]

**Pull request merged**